docs/install/users.rst

```diff
@@ -1,33 +1,43 @@
 .. _install/users:
 
-Creating Users on the host
-==========================
+Creating users and user groups on the host
+==========================================
 
 .. note::
-  By default the ``site.yml`` playbook does not create any users on the host machine.
+  By default the ``site.yml`` playbook does not create any users nor user groups on the host machine.
 
-  This step is optional because in some scenarios users might already exist on the host machine
+  This step is optional because in some scenarios users and user groups might already exist on the host machine
   and don't need to be created.
 
 .. _install/users-playbook:
 
 Using the users playbook
 ------------------------
 
-The ``ansible/`` directory contains a ``users.yml`` playbook that makes it easier to create new users on the host in batches.
+The ``ansible/`` directory contains a ``users.yml`` playbook that makes it easier to create new users and user groups on the host in batches.
 
 First you need to create a new ``users-config.yml`` with the following content:
 
 .. code-block:: yaml
 
+    user_groups:
```

Comment:
Should it be `user_groups` or `groups`?

(no strong feeling, it's more about succinctness)

Reply:
I tried but `groups` is a variable already [taken](https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html) by Ansible. It took me sometime to realize this ;-)